src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Logic/Add-ArmWorkflowHttpAction.ps1

```diff
@@ -0,0 +1,34 @@
+function Add-ArmWorkflowHttpAction {
```

Comment:
I don't think we want to create a cmdlet for each possible action. I suggest we make it a way more generic. WorkflowDefinition should be considered as data and not structure. I suggest we take in a json object as the input of the New-ArmLogicApp function